datafusion/core/tests/dataframe/dataframe_functions.rs

```diff
@@ -111,7 +111,7 @@ async fn test_fn_ascii() -> Result<()> {
 
 #[tokio::test]
 async fn test_fn_bit_length() -> Result<()> {
-    let expr = bit_length(col("a"));
+    let expr = bit_length(vec![col("a")]);
```

Comment:
since `bit_length` only ever takes a single parameter, I think it would be better to keep this as a single `expr` argument (to keep the UX nicer to call `bit_length`).

If the function can take potentially different numbers of parameters, that is when `Vec<Expr>` is more appropriate I think

Reply:
https://github.com/apache/arrow-datafusion/pull/9782/commits/17d723834338f8165168c3df8e105f3802d84e87